src/preprocessing/passes/bv_to_int.h

```diff
@@ -73,6 +73,7 @@
 #include "context/cdhashset.h"
 #include "preprocessing/preprocessing_pass.h"
 #include "theory/arith/nl/iand_utils.h"
+#include "util/rational.h"
```

Comment:
Rational doesn't seem to be used in this header, include shouldn't be needed here.